test/jdk/java/awt/TrayIcon/TrayIconPopup/TrayIconPopupTest.java

```diff
@@ -45,7 +45,7 @@ public class TrayIconPopupTest {
 
     boolean actionPerformed = false;
     Object actionLock = new Object();
-    static final int ATTEMPTS = 50;
+    static final int ATTEMPTS = 10;
```

Comment:
Each attempt sleeps for 3 seconds, so 50 attempts is way too many and exceeds the default test timeout.

Reply:
OK. But I'm not seeing how this change helps the test pass.

Reply:
It does not, but when the test does fail, it takes ages to fail.
(e.g. if we are missing some env variable which we are rely on to disable system tray support when it is [needed](https://bugs.openjdk.org/browse/JDK-8322750)).